tools/hangar/src/test_corpus/valid/bucket_events.w.test.ts

```diff
@@ -0,0 +1,12 @@
+// This file is generated by tools/hangar/src/generate_tests.ts
```

Comment:
I don't think this file needs to be committed to git

Reply:
I figured it would be ignored by [this line](https://github.com/winglang/wing/blob/f22671880cf9c94505623a18a6f5c97dc01fb92c/.gitignore#L48) so I'm not sure how it got added, but you could try adding another gitignore inside `tools/hangar` to see if that fixes it

Reply:
```
git rm -rf --cached .
git add .
```
worked for me :)